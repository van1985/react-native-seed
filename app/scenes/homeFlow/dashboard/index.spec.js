import React from 'react';
import renderer from 'react-test-renderer';
import Dashboard from './index';

test('renders correctly', () => {
  const tree = renderer.create(<Dashboard />).toJSON();
  expect(tree).toMatchSnapshot();
});
