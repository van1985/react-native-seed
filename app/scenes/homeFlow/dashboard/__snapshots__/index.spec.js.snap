// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#efefef",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#FFFFFF",
        "flex": 1,
        "margin": 50,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../app/assets/background-birthday.png",
        }
      }
      style={
        Object {
          "height": 120,
          "width": 280,
        }
      }
    />
    <View
      style={
        Object {
          "flexDirection": "row",
          "flexWrap": "wrap",
          "justifyContent": "space-between",
          "marginTop": 20,
          "paddingLeft": 20,
          "paddingRight": 20,
        }
      }
    >
      <Image
        source={
          Object {
            "testUri": "../../../app/assets/silhouette200.png",
          }
        }
        style={
          Object {
            "borderRadius": 25,
            "height": 50,
            "width": 50,
          }
        }
      />
      <Image
        source={
          Object {
            "testUri": "../../../app/assets/silhouette200.png",
          }
        }
        style={
          Object {
            "borderRadius": 25,
            "height": 50,
            "width": 50,
          }
        }
      />
      <Image
        source={
          Object {
            "testUri": "../../../app/assets/silhouette200.png",
          }
        }
        style={
          Object {
            "borderRadius": 25,
            "height": 50,
            "width": 50,
          }
        }
      />
      <Image
        source={
          Object {
            "testUri": "../../../app/assets/silhouette200.png",
          }
        }
        style={
          Object {
            "borderRadius": 25,
            "height": 50,
            "width": 50,
          }
        }
      />
    </View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "backgroundColor": "transparent",
          "color": "#FFFFFF",
          "fontSize": 25,
          "left": 120,
          "position": "absolute",
          "top": 20,
        }
      }
    >
      May
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "backgroundColor": "transparent",
          "color": "#FFFFFF",
          "fontSize": 40,
          "fontWeight": "bold",
          "left": 125,
          "position": "absolute",
          "top": 50,
        }
      }
    >
      7
    </Text>
  </View>
</View>
`;
